include/mage.hpp

```diff
@@ -0,0 +1,2780 @@
+// Copyright 2022 Memgraph Ltd.
```

Comment:
@antepusic why is there `mage.hpp` and `mgp.hpp` shouldn't that be one file (probably the same applies to `mg_exceptions.hpp`)?